Drakhtar/SDLAudioManager.cpp

```diff
@@ -120,7 +131,7 @@ void SDLAudioManager::playMusic(int tag, int loops) {
 	}
 }
 
-int SDLAudioManager::setMusicVolume(int volume) {
+int SDLAudioManager::setMusicVolume(int volume) { // volume = 2 is quite low already, play with that number 
```

Comment:
What does "play with that number" exactly mean?

Reply:
It means that if volume is 2, the sound is low, so based on that, anything higher than the value "2" will be higher volume.
In short, if volume is 2, you will barely hear the music